WaarpR66/src/main/java/org/waarp/openr66/client/AbstractTransfer.java

```diff
@@ -377,7 +389,8 @@ protected static boolean getParams(String[] args, boolean submitOnly) {
         return true;
       } catch (final WaarpDatabaseException e) {
```

Comment:
And what about something like that to make it more clear and explicit (without having to read the stacktrace):

```suggestion
      } catch (final WaarpDatabaseNoDataException e) {
          logger.error(Messages.getString("No transfer found with this id and partner");
          return false
      } catch (final WaarpDatabaseException e) {
```

Reply:
Agree